Cargo.toml

```diff
@@ -16,7 +16,9 @@ exclude = [
 
 [dependencies]
 rand = "0.3"
-arrayref = "0.3.2"
+
+# Support for #![no_std] is only on master right now
+arrayref = { git = "https://github.com/droundy/arrayref.git" }
```

Comment:
Might want to wait for another release of `arrayref` before merging this /cc @droundy